### PR TITLE
ci(*): add actions/labeler to label automatically to recognize scope of Pull Request easily

### PR DESCRIPTION
**Pull request by manudeli**
This [actions/labeler](https://github.com/actions/labeler) will label what package scope is on each Pull Request automaticallly. After this Pull Request merged, labeler can know what package scope have been changed by directory
This action is supported by GitHub Actions officially too like [actions/setup-node](https://github.com/actions/setup-node)

### [Example screenshot](https://github.com/suspensive/react/pulls?q=is%3Apr+is%3Aclosed)

image

### Expectation

I am expecting each maintainers can recognize each Pull Request's package scope by labels of it more easily.

### CODE REVIEW

**Review by fh-nabeel-ahmad — COMMENTED** [review]
LGTM

**Comment by TkDodo** [comment]
we already have labels for packages:

Screenshot 2024-03-03 at 10 45 29

will they be re-used ?

**Comment by manudeli** [comment]
> we already have labels for packages:
> 
> Screenshot 2024-03-03 at 10 45 29
> 
> will they be re-used ?

Oh thanks, I'll update it

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+'package: svelte-query':
+  - 'packages/svelte-query/**/*'
+'package: svelte-query-devtools':
+  - 'packages/svelte-query-devtools/**/*'
+'package: svelte-query-persist-client':
+  - 'packages/svelte-query-persist-client/**/*'
+'package: vue-query':
+  - 'packages/vue-query/**/*'
+'package: vue-query-devtools':
+  - 'packages/vue-query-devtools/**/*'
```
Comment (manudeli) [inline .github/labeler.yml:44]: @TkDodo I updated them as following your guide

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+'documentation':
+  - 'docs/**/*'
```
Comment (manudeli) [inline .github/labeler.yml:46]: I found documentation label in this repository. so I named as documentation

**Review by manudeli — COMMENTED** [review]
@TkDodo I updated now

**Pull request merged**